src/components/include/llprint.h

```diff
@@ -5,27 +5,29 @@
 #include <stdio.h>
 #include <string.h>
 
+#include <cos_component.h>
+
 static void
 cos_llprint(char *s, int len)
 {
 	call_cap(PRINT_CAP_TEMP, (int)s, len, 0, 0);
 }
 
-int
+static size_t
 prints(char *s)
 {
-	int len = strlen(s);
+	size_t len = strlen(s);
 
 	cos_llprint(s, len);
 
 	return len;
 }
 
-int __attribute__((format(printf, 1, 2))) printc(char *fmt, ...)
+static size_t __attribute__((format(printf, 1, 2))) printc(char *fmt, ...)
```

Comment:
strange that the auto-formatter didn't catch this (`printc` should be on the next line)